### PR TITLE
Add odh + opf intro

### LINKED ISSUE #76: Short What is ODH + OPF doc
We need a short doc (1 paragraph max for each items with links out to their project pages) to explain the environment we develop in.

### PR DESCRIPTION
**Pull request by aakankshaduggal**
Fixes #76

### CODE REVIEW

**Review by oindrillac — CHANGES_REQUESTED** [review]
let's add some additional context for data science users

**Review by oindrillac — APPROVED** [review]
/lgtm 👍

File: docs/odh_and_opf.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+
+# What is Operate First?
+*“A concept to incorporate operational experience into software projects.”*
+
+[Operate First](https://www.operate-first.cloud/) is an initiative to “operate” software in a production-grade environment - bringing users, developers and operators closer together. It is a concept where open source developers bring their projects to a production cloud during development where they can interact with a cloud provider’s operators and gain valuable feedback on the operational considerations of their project.  It uses the same community-building process of open-source projects, but extended to ops procedures and data. 
+
+Operate First manages a ready-to-use deployment of Open Data Hub on the [Mass Open Cloud](https://massopen.cloud/)(MOC) Smaug cluster which is available for the public to experience. [Here](https://www.operate-first.cloud/apps/content/odh/README.html) is a list of components that are currently available for anybody to start using.
```
Comment (MichaelClifford) [inline docs/odh_and_opf.md:11]: Take out reference to `Smaug` as the specific cluster name will change in the future.

**Review by MichaelClifford — CHANGES_REQUESTED** [review]
Nice :100: 
one small edit

File: docs/odh_and_opf.md (hunk 1 above)
Comment (aakankshaduggal) [reply docs/odh_and_opf.md]: Done :)

**Pull request merged**